MPF.Core/UI/ViewModels/MainViewModel.cs

```diff
@@ -160,6 +161,51 @@ public string OutputPath
         }
         private string _outputPath;
 
+        /// <summary>
+        /// Currently provided output path, with < > variables evaluated
+        /// Not guaranteed to be a valid path
+        /// </summary>
+        public string OutputPathEvaluated
```

Comment:
If this is getting unwieldly, feel free to make this property invoke a method instead of a getter.